crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py

```diff
@@ -0,0 +1,370 @@
+                                          'status of the lots '
+                                          'loaded')
+    lines_id = fields.Many2many('stock.production.lot',
+                                string='Invoice Lines to Select',
+                                help='Field used to load the ids of '
+                                     'invoice lines in invoices writed')
+    scan_data = fields.Text('Products',
+                            help='Field used to load and show the '
+                            'products')
+    scaned_data = fields.Text('Products',
+                              help='Field used to load the ids of '
+                              'products loaded')
+    current_status = fields.Text('Status',
+                                 help='Field used to show the current '
+                                 'status of the product '
+                                 'loaded(Name and quantity)')
+
+    @api.multi
+    def get_metasearch_view_brw(self):
+        """
+        @return: view with metasearch field
+        """
+        view_id = self.env.\
+            ref('crm_rma_lot_mass_return.view_enter_product')
+        return view_id
+
+    @api.multi
+    def render_metasearch_view(self):
+        """
+        Render wizard view with metasearch field
+        """
+        view = self.get_metasearch_view_brw()
+        if view:
+            return {
+                'name': _('Search Product'),
+                'type': 'ir.actions.act_window',
+                'view_type': 'form',
+                'view_mode': 'form',
+                'res_model': 'returned.lines.from.serial.wizard',
+                'view_id': view.id,
+                'views': [(view.id, 'form')],
+                'target': 'new',
+                'res_id': self.id,
+            }
+
+    @api.model
+    def get_data_of_products(self, input_data):
+        data_line = []
+        for item in input_data and input_data.split('\n') or []:
+            if '*' in item:
+
+                comput = item.split('*')
+                data_it_1 = '0'
+                data_it_2 = ''
+                if len(comput) >= 2:
+                    data_it_1 = comput[1]
+                if len(comput) >= 3:
+                    data_it_2 = comput[2]
+                if comput[0]:
+                    data_line.append((comput[0], (data_it_1, data_it_2)))
+            else:
+                if item.strip():
+                    data_line.append((item.strip(), ('0', '' )))
+        return data_line
+
+    def _get_lots_from_scan_data(self, input_data):
+
+        input_lines = self.get_data_of_products(input_data)
+        invoice = self.env['account.invoice']
+        lot = self.env['stock.production.lot']
+
+        prodlot_set_ids = set()
+        lot_set_ids = set()
+        invoice_lot_set_ids = set()
+        lots_lot_set_ids = set()
+
+        if not input_lines:
+            return False, False
```

Comment:
@osvalr There is a bug here   
```
elements_searched, line_found, lot_lots_ids = \
ValueError: need more than 2 values to unpack
```
You cannot return two values when waiting three values.
Fix it, please.